### PR TITLE
Add example code that uses some fixed plots and AnyPlot

### PR DESCRIPTION
**Pull request by PeterC-DLS**

### CODE REVIEW

File: client/example/src/App.tsx [hunk 1]
```diff
@@ -25,29 +43,113 @@ class AppMain extends React.Component<AppMainProps, AppMainStates> {
...
+      [3, 2]
+    ) as NDT;
+    const heatmapProps = {
+      selections: [] as SelectionBase[],
+      plotConfig: {
+        title: 'Sample Heatmap Plot',
+        xLabel: 'x-axis',
+        yLabel: 'y-axis',
+      } as PlotConfig,
+      key: 'Example heatmap',
```
Comment (VictoriaBeilsten-Edmands) [inline client/example/src/App.tsx:86]: Remove `key`

**Review by VictoriaBeilsten-Edmands — COMMENTED** [review]
The tabs and plots are working for me. The selections don't work in the demos, but there is also a warning in the console about WebGL context being lost.

File: client/example/src/App.tsx (hunk 1 above)
Comment (PeterC-DLS) [reply client/example/src/App.tsx:86]: I think removing `key` should be in another PR - it's unnecessary and unused for all bar the line plots.

**Comment by PeterC-DLS** [comment]
That WebGL context loss seems to be a regular thing - not sure if it's harmful...

File: client/example/src/App.tsx (hunk 1 above)
Comment (PeterC-DLS) [reply client/example/src/App.tsx]: Done

**Review by VictoriaBeilsten-Edmands — APPROVED** [review]

**Pull request merged**